.gitignore

```diff
@@ -12,6 +12,7 @@ node_modules
 # local env files
 .env.local
 .env.*.local
+.python-version
```

Comment:
up for feedback if you want to just commit this file vs adding it in during the vagrant provisioning

Reply:
I'm on board with how you've handled. It's more related to how we've chosen to set up the python environment, and not really part of the project source itself, so this is good.